app/javascript/styles/components/Calendar.scss

```diff
@@ -165,3 +171,7 @@
     margin-top: 3rem;
   }
 }
+
+.calendar-grid__cell {
+  width: 250px;
```

Comment:
Should we add a media-query to this or change the grid to 1 column at the mobile size? This seems to make less text read on screen for me:


### After this change
<img width="382" alt="image" src="https://user-images.githubusercontent.com/565647/71599601-885a0880-2b19-11ea-968d-463072ac51fc.png">

### Before the Change
<img width="382" alt="image" src="https://user-images.githubusercontent.com/565647/71599616-94de6100-2b19-11ea-9517-8d5bf62bf5bd.png">

Reply:
Huh, I was thinking it was a bit too convenient for mobile to just be showing a scaled-down three column layout. I'll take another look at this.